CMakeLists.txt

```diff
@@ -932,12 +933,11 @@ set(QGIS_INSTALL_SYS_LIBS TRUE CACHE BOOL "If set to TRUE install all required s
 
 #############################################################
 # Python build dependency
-
-find_package(PythonLibrary REQUIRED)
-set(MIN_PYTHON_VERSION "3.7")
-if(${PYTHON_SHORT_VERSION} VERSION_LESS ${MIN_PYTHON_VERSION})
-    message(FATAL_ERROR "Python version ${PYTHON_SHORT_VERSION} is too old. Minimum Python version is ${MIN_PYTHON_VERSION}.")
-endif()
+set(Python_FIND_FRAMEWORK "LAST")
+find_package(Python 3.7 REQUIRED COMPONENTS Interpreter)
```

Comment:
FindPython.cmake is part of cmake from CMAKE 3.12. minimm required is 3.10 https://github.com/qgis/QGIS/blob/41fbcbe7ceae64389567ea3d4ad8fcd535c1284e/CMakeLists.txt#L3

Reply:
fixed